specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/HybridCompute.json

```diff
@@ -0,0 +1,3782 @@
+          "description": "The resource id of the parent cluster (Azure HCI) this machine is assigned to, if any."
+        },
+        "mssqlDiscovered": {
+          "type": "string",
+          "description": "Specifies whether any MS SQL instance is discovered on the machine."
+        },
+        "detectedProperties": {
+          "$ref": "#/definitions/DetectedProperties",
+          "description": "Detected properties from the machine."
+        },
+        "networkProfile": {
+          "$ref": "#/definitions/NetworkProfile",
+          "description": "Information about the network the machine is on."
+        }
+      },
+      "description": "Describes the properties of a hybrid machine."
+    },
+    "MachineUpdateProperties": {
+      "type": "object",
+      "properties": {
+        "locationData": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/locationData"
+        },
+        "osProfile": {
+          "$ref": "#/definitions/OSProfile"
+        },
+        "cloudMetadata": {
+          "$ref": "#/definitions/CloudMetadata",
+          "description": "The metadata of the cloud environment (Azure/GCP/AWS/OCI...)."
+        },
+        "agentUpgrade": {
+          "$ref": "#/definitions/AgentUpgrade",
+          "description": "The info of the machine w.r.t Agent Upgrade"
+        },
+        "parentClusterResourceId": {
+          "type": "string",
+          "description": "The resource id of the parent cluster (Azure HCI) this machine is assigned to, if any."
+        },
+        "privateLinkScopeResourceId": {
+          "type": "string",
+          "description": "The resource id of the private link scope this machine is assigned to, if any."
+        }
+      },
+      "description": "Describes the ARM updatable properties of a hybrid machine."
+    },
+    "Machine": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/MachineProperties",
+          "description": "Hybrid Compute Machine properties"
+        },
+        "resources": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MachineExtension"
+          },
+          "description": "The list of extensions affiliated to the machine"
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/Identity"
+        },
+        "kind": {
+          "$ref": "#/definitions/PrivateCloudKind"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Describes a hybrid machine."
+    },
+    "PrivateCloudKind": {
+      "type": "string",
+      "description": "Indicates which kind of VM fabric the instance is an instance of, such as HCI or SCVMM etc.",
```

Comment:
I do not quite understand this statement. Can we make it simpler? 

"Indicate which kind of Arc machine placement on-premises, such as HCI, SCVMM or VMware etc.",